Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM centos:5
+
+RUN yum update  -y 
+RUN yum install -y wget
+RUN wget http://download.fedoraproject.org/pub/epel/5/x86_64/epel-release-5-4.noarch.rpm
+RUN rpm -ivh epel-release-5-4.noarch.rpm
+
+RUN yum update -y  && yum install -y \
```

Comment:
yum update is already done at line 3. You could also add wget to this list instead of doing it at line 4.